en/mkdocs.yml

```diff
@@ -42,7 +42,6 @@ theme:
   feature:
     tabs: true
   language: en
-
 # Navigation
```

Comment:
might need to add the following plugin to the plugins section:

plugins:
  - include-markdown

Reply:
worked without the plugin locally. anyway added the plugin in [69b7592](https://github.com/wso2/open-healthcare-docs/pull/28/commits/69b75929cc69eb952cd42365740b98a282b368d4)